exonum-java-binding/service-archetype/src/main/resources-filtered/archetype-resources/__rootArtifactId__-service/pom.xml

```diff
@@ -52,13 +52,23 @@
       <scope>provided</scope>
     </dependency>
 
-    <!-- Override the scope of exonum-messages in case \${rootArtifactId}-messages depends on them -->
+    <!--
+     Override the default 'compile' scope of \${rootArtifactId}-messages dependencies
+     that are provided by the framework (see the explanation above): exonum-messages
+     and protobuf-java.
+    -->
     <dependency>
       <groupId>com.exonum.messages</groupId>
       <artifactId>exonum-messages</artifactId>
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
```

Comment:
That's a consequence of creating a separate module for the messages:
- to be usable by the clients, it shall declare its dependencies in 'compile' (default) scope
- the service has to override the dependencies on 'framework-provided' modules to 'provided'